### ISSUE TITLE
research ideas of 5 september 2020

### ISSUE THREAD

**Issue by SonixApache** [d3athrow/vgstation13#27737]
[discussion] this is a copypaste of all the shit said in discord, i will sort it out some time soon

hinge [Tempbanned]Today at 8:19 AM
i'll say it again and forever, sci should be a mix of actual research, chemistry, and making experiments
subject weird shit to chemicals and energy fields to get data, use data to research shit
hinge [Tempbanned]Today at 8:20 AM
hmmm looks like this plasma yadda bullshit doesn't respond to this, what if i make [analysis fluid X] and try it then?
hinge [Tempbanned]Today at 8:21 AM
I really like how system shock 2 did it
different items need different chemical mixtures to research

SonixApacheToday at 8:20 AM
science should be prototypes, randomization and liberal use of plasma

hinge [Tempbanned]Today at 8:21 AM
give sci a brand new chem machine that has all the weird out there elements the normal one doesn't
then make them make new weird chems to analyze shit with
SonixApacheToday at 8:22 AM
we've got a lot of chems already...
hinge [Tempbanned]Today at 8:22 AM
shit like francium, curium, etc
SonixApacheToday at 8:22 AM
could get some weirder ones from xenoarch
and/or engineering
yo actually that'd be cool as fuck
if the rad collectors for the various engines produces various kinds of weird substances due to plasma

hinge [Tempbanned]Today at 8:23 AM
"if I surround this artifact in [xenon] gas, and THEN irradiate it, what happens?"
xenobio should be about capturing alien life and dissecting it, and analyzing the organs
not slime ranching

Anti-ERP Metaclub KhanToday at 8:23 AM
it would be based but extremely jobbyfied if science was done through actual experiments and data
hinge [Tempbanned]Today at 8:23 AM
YES
SonixApacheToday at 8:23 AM
thing is, rounds are really fucking short
hinge [Tempbanned]Today at 8:23 AM
i want the jobby in sci
SonixApacheToday at 8:23 AM
we have to take that into account
Anti-ERP Metaclub KhanToday at 8:24 AM
yeah i suppose
hinge [Tempbanned]Today at 8:24 AM
it shouldn't be much jobbier than doing med
SonixApacheToday at 8:24 AM
also full randomization is fucking gay, so there'd have to be patterns
one interesting thought i had was using the persistence subsystem to save discoveries for some time
hinge [Tempbanned]Today at 8:24 AM
and with any hope it should be fun to do
maybe make different research shit have different tendencies
"oh, medical research tends to respond better to [x]"
or "things to do with minerals respond well to [list of analysis chems] and [X experiments]

Anti-ERP Metaclub KhanToday at 8:24 AM
what if science started with the tools to make high tier machines at round start but through experiments you got powertools and upgrades to objects?

SonixApacheToday at 8:25 AM
we could relegate machine parts and upgrades to mechanics in their entirety
imo we would throw the entire current research system out the window
mechanics can get their own little progression system to unlock better parts for machines or something

hinge [Tempbanned]Today at 8:26 AM
how i was sorta thinking it would go is doing the experiments gives you "data" which can be spent to boost relative research or something
SonixApacheToday at 8:26 AM
i'll open a git issue
oh, that's sorta how astroneers does it
simplified
hinge [Tempbanned]Today at 8:27 AM
doing experiments better gives more data to spend
SonixApacheToday at 8:27 AM
you find some weird shit, throw it in a research bank, then use the research 'bits' on stuff you want

**Comment by Buchse** [comment]
TL:DR, the idea is to make science function around experiments and such rather than playing point and click. 
The flow of research as I was thinking would go something like this:
>Scientist A gets a sample of something, could be a rock, could be weird meat.
>Scientist A decides to research it
>He goes to the Science Chem Machine, which would have all the weird elements the normal one doesn't have, and mixes up some analysis fluid that corresponds to dealing with biological stuff
>He applies to the chemcial to the meat, or fills a jar with it and sticks the meat in, however, and then for example, subjects it to electricity or radiation
>The experiment, using that specific material, analysis fluid, and test it's subjected to, gives a certain amount of data to be collected. Since he's looking at meat, that data would be [biological].
>Scientist A now has [X] amount of biological data, which he can spend towards making something connected to cloning, or hydroponics, or however he likes. 

Science would become a matter of finding the best experiments to perform on certain specimens of certain types, to get the greatest amount of data from them. It'd be a bit like figuring out the best cryomix for a certain job, or getting an atmos ratio how you like it. 

Data would be basically an abstract resource, having 2-3 different kinds of it, like (materials, energy, biology), which different tech would cost in different amounts. Something like an advanced brute kit might need both materials and biology, but advanced organs would need all three to obtain.

**Comment by west3436** [comment]
The rot consumes
